recipes/fwdpy11/meta.yaml

```diff
@@ -24,13 +24,17 @@ requirements:
     - tskit >=0.1.5
     - gsl
     - openblas
+    - attrs
+    - setuptools_scm
   run:
     - python
     - numpy >=1.10
     - tskit >=0.1.5
     - gsl
     - openblas
     - sparse
+    - attrs
+    - black
```

Comment:
Just curious: I `black` really required as a `run` dependency? Seems rather unusual, is all.

Reply:
Yes, it is used to provide a method to pretty-print string representations of classes.

Reply:
Ah, okay -- thanks for clarifying!